README.rst

```diff
@@ -41,14 +41,20 @@ Primary use cases include:
    up with a central database (to download new content and upload
    analytics) when in an area with internet connectivity.
 
-We would love for you to get involved!
---------------------------------------
+Get involved!
+-------------
 
 -  Learn how you can contribute code on our `KA Lite GitHub Wiki <https://github.com/learningequality/ka-lite/wiki>`__
 -  Report bugs by `creating issues <https://github.com/learningequality/ka-lite/wiki/Report-Bugs-by-Creating-Issues>`__
 -  Read more about the project's motivation at `Introducing KA Lite, an offline version of Khan
    Academy <http://jamiealexandre.com/blog/2012/12/12/ka-lite-offline-khan-academy/>`__.
 
+Connect
+^^^^^^^
+
+- IRC: **#kalite** on Freenode or `our Slack room #kalite <https://learningequality.slack.com/archives/kalite>`__ (both are connected)
```

Comment:
I think you need (by default) a learningequality Google email to use the Slack room - might be best to just point to the IRC Channel?